src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -1210,6 +1219,18 @@ public function count(): int
         return \count($this->defaults);
     }
 
+    /**
+     * Sets whether ignore undefined options.
+     *
+     * @return $this
+     */
+    public function ignoreUndefined(bool $ignore = true): static
```

Comment:
As we discussed previously, we should keep the `set` prefix here for consistency with all setter methods in this class.